src/plugins/tabs/components/tabs.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div class="c-tabs-view">
+        <div class="c-tabs-view__tabs-holder c-compact-button-holder">
+            <button class="c-tabs-view__tab c-compact-button"
+                v-for="(tab,index) in tabsList"
+                :key="index"
+                :class="[
+                    {'is-current': tab=== currentTab}, 
+                    tab.type.definition.cssClass
+                ]"
+                @click="showTab(tab)">
+                <span class="c-button__label">{{tab.model.name}}</span>
+            </button>
+            <div style="position: absolute; top: 0; left: 0; background: #009ad0; min-height: 100%; min-width: 100%; font-size: 20px; text-align: center;"
```

Comment:
@dtailor90 changes look good to me. I have one minor comment. Is there a reason the styles are not defined as a css class? would be cleaner to not mix the styles with the markup.

Reply:
@psarram - Good catch! But i left this here as a place holder, @charlesh88 is going to take a final look at it and fix the styling.